code/game/objects/items/stacks/stack.dm

```diff
@@ -207,10 +207,33 @@
 			to_chat(usr, "<span class='warning'>There is another [R.title] here!</span>")
 			return FALSE
 
+		if(R.one_per_turf)
+			var/list/oneperturf = list(
+				/obj/structure/girder,
+				/obj/structure/falsewall,
+				/obj/structure/window,
+				/obj/structure/cult,
+				/obj/structure/table,
+				/obj/structure/chair,
+				/obj/structure/bed,
+				/obj/machinery/computer,
+				/obj/structure/computerframe,
+				/obj/machinery/door,
+				/obj/machinery/porta_turret
+			)
+			var/obj/S = locate() in get_turf(src)
+			if(is_type_in_list(S, oneperturf))
+				to_chat(usr, "<span class='warning'>There is already som,e here!</span>")
```

Comment:
What were you trying to write here?

Reply:
"something" Fixed now, was sure I checked that, but appearantly not. Thanks for looking through this!